Meta/build-image-qemu.sh

```diff
@@ -39,6 +39,10 @@ INODE_COUNT=$((INODE_COUNT + 2000))  # Some additional inodes for toolchain file
 DISK_SIZE_BYTES=$((($(disk_usage "$SERENITY_SOURCE_DIR/Base") + $(disk_usage Root) ) * 1024 * 1024))
 DISK_SIZE_BYTES=$((DISK_SIZE_BYTES + (INODE_COUNT * INODE_SIZE)))
 
+# At the time of writing, in the default path DISK_SIZE_BYTES ~= 1.4 GiB and INODE_COUNT ~= 140k.
+# Let's use this ratio of bytes per inode instead of manually trying to guess the number of inodes.
+BYTES_PER_INODE=11264
```

Comment:
Isn't it exactly one inode per file? Can we just count the number of files, with `find | wc -l` or something?

I suppose we want to have room for additional files in addition to the ones used for the system.

Eh, I suppose any heuristic that works is fine. Maybe you could say in the commit message why the current heuristic wasn't working.